src/AppBundle/Command/DeleteUserCommand.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Doctrine\Common\Persistence\ObjectManager;
+
+/**
+ * A command console that deletes users and from the database.
+ * To use this command, open a terminal window, enter into your project
+ * directory and execute the following:
+ *
+ *     $ php app/console app:delete-user
+ *
+ * To output detailed information, increase the command verbosity:
+ *
+ *     $ php app/console app:delete-user -vv
+ *
+ * See http://symfony.com/doc/current/cookbook/console/console_command.html
+ *
+ * @author Oleg Voronkovich <oleg-voronkovich@yandex.ru>
+ */
+class DeleteUserCommand extends ContainerAwareCommand
+{
+    const MAX_ATTEMPTS = 5;
+
+    /**
+     * @var ObjectManager
+     */
+    private $em;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            // a good practice is to use the 'app:' prefix to group all your custom application commands
+            ->setName('app:delete-user')
+            ->setDescription('Deletes users from the database')
+            ->setHelp($this->getCommandHelp())
+            // commands can optionally define arguments and/or options (mandatory and optional)
+            // see http://symfony.com/doc/current/components/console/console_arguments.html
+            ->addArgument('userIdentity', InputArgument::OPTIONAL, 'The username or email of an existing user')
```

Comment:
@javiereguiluz, what concerns you? Do you think it's a bad practice? We use a single class for few different things (AppBundle\Twig\AppExtension), and it's ok :)